compiler/src/org.graalvm.compiler.nodes/src/org/graalvm/compiler/nodes/extended/RawStoreNode.java

```diff
@@ -58,21 +58,28 @@
     @Input ValueNode value;
     @OptionalInput(State) FrameState stateAfter;
     private final boolean needsBarrier;
+    private boolean isVolatile;
 
     public RawStoreNode(ValueNode object, ValueNode offset, ValueNode value, JavaKind accessKind, LocationIdentity locationIdentity) {
-        this(object, offset, value, accessKind, locationIdentity, true);
+        this(object, offset, value, accessKind, locationIdentity, true, false, null, false);
     }
 
-    public RawStoreNode(ValueNode object, ValueNode offset, ValueNode value, JavaKind accessKind, LocationIdentity locationIdentity, boolean needsBarrier) {
-        this(object, offset, value, accessKind, locationIdentity, needsBarrier, null, false);
+    public RawStoreNode(ValueNode object, ValueNode offset, ValueNode value, JavaKind accessKind, LocationIdentity locationIdentity, boolean isVolatile) {
```

Comment:
There slipped a tiny bug through: A few lines down is a node intrinsic that used this ctor with needsbarrier and this commit changed it to isVolatile, this caused some of our internal tests to fail. I fixed that internally and started the integration process.

Reply:
Thanks for fixing that one. And thanks for the quick turnaround on that one.